interfaces/builtin/opengl_test.go

```diff
@@ -123,9 +115,20 @@ func (s *OpenglInterfaceSuite) TestUsedSecuritySystems(c *C) {
 	apparmorSpec := &apparmor.Specification{}
 	err := apparmorSpec.AddConnectedPlug(s.iface, s.plug, nil, s.slot, nil)
 	c.Assert(err, IsNil)
-	c.Assert(apparmorSpec.SecurityTags(), DeepEquals, []string{"snap.client-snap.app-accessing-opengl"})
-	aasnippet := apparmorSpec.SnippetForTag("snap.client-snap.app-accessing-opengl")
-	c.Assert(aasnippet, Equals, expectedSnippet)
+	c.Assert(apparmorSpec.SecurityTags(), DeepEquals, []string{"snap.consumer.app"})
```

Comment:
Ah, perhaps I missed that. My whole point was to have per-backend tests methods as this makes them, in my experience, easier to write and understand.